import-export-cli/utils/constants.go

```diff
@@ -86,6 +86,7 @@ const LogPrefixError = "[ERROR]: "
 
 // String Constants
 const SearchAndTag = "&"
+const APISecurityMutualSsl = "mutualssl"
```

Comment:
May use the constant key's ssl part in ALL CAPS, as that is an abbreviation like API.

Reply:
Fixed via https://github.com/wso2/product-apim-tooling/pull/594